data-r2dbc/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     implementation mn.micronaut.json.core
     compileOnly mnSerde.micronaut.serde.oracle.jdbc.json
     implementation mnReactor.micronaut.reactor
+    implementation libs.managed.reactor.netty.core
 
```

Comment:
Similar to https://github.com/micronaut-projects/micronaut-r2dbc/pull/655/files#r1670652324